src/deepmol/feature_selection/base_feature_selector.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, feature_selector):
         """
         Initialize the feature selector.
         """
+        self.inplace = None
```

Comment:
I do not understand the reasons for the changes made in this file... The `select_features` method can be inplace or not, the `_transform` should never be inplace, right?

Reply:
Did this fail in something that you tried?